resource-managers/yarn/src/test/scala/org/apache/spark/network/yarn/YarnShuffleServiceSuite.scala

```diff
@@ -403,7 +403,9 @@ class YarnShuffleServiceSuite extends SparkFunSuite with Matchers with BeforeAnd
     val metrics = metricSetRef.get(metricsSource).asInstanceOf[MetricSet].getMetrics
 
     assert(metrics.keySet().asScala == Set(
+      "blockTransferRate",
```

Comment:
Why not to use the same trick "// Use sorted Seq instead of Set for easier comparison when there is a mismatch" here?

Reply:
Updated here and in `ExternalShuffleServiceMetricsSuite`